### DISCUSSION TITLE
Observatory

### DISCUSSION THREAD

**Discussion by FightingForFun** [XTLS/Xray-core#4072] [General]
How to set up using Observatory and probeUrl - https://www.google.com/generate_204 so that the server is considered operational with a 204 response, not as it is now with a 200 response?

The outbound server returns me a login/password for the web panel, but xray considers the server to be operational.

Burst is not suitable for me... because I have ~50 servers, and it overloads my old processor with simultaneous checks.

**Comment by lambdaxyzt** [comment]
check out https://github.com/XTLS/Xray-core/discussions/3138
